test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -1209,6 +1202,10 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 						"p1": "v1",
 						"p2": "v2",
 					},
+					RetryStrategy: &kueue.ProvisioningRequestRetryStrategy{
+						BackoffLimitCount:  ptr.To(int32(1)),
+						BackoffBaseSeconds: ptr.To(int32(1)),
```

Comment:
```suggestion
						BackoffLimitCount:  ptr.To[int32](1),
						BackoffBaseSeconds: ptr.To[int32](1),
```

There are the same comments in other places.

Reply:
Done